src/functions-and-arrays.js

```diff
@@ -78,7 +113,7 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(wordArr, word) {}
 
```

Comment:
You seem to have found the tasks difficult. Please ask us if you want us (Marian, Karina or me, Joanne) to explain certain things to you! :)